src/main/java/com/example/controller/MainMenuController.java

```diff
@@ -5,12 +5,25 @@
 import javafx.beans.value.ChangeListener;
 import javafx.beans.value.ObservableValue;
 import javafx.fxml.FXML;
+import javafx.scene.control.MenuItem;
 import net.rgielen.fxweaver.core.FxmlView;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+
 
 @Component
 @FxmlView("/view/mainMenuView.fxml")
 public class MainMenuController implements ChangeListener<ApplicationState> {
+    @Autowired
+    GraphController graphController;
+    @FXML
+    MenuItem importButton;
```

Comment:
Instead of creating buttons here. better option would be to move them to separate controller ex. FileMenuController. Similar approach is applied in DrawMenuController for example.